src/dbnode/storage/series/buffer.go

```diff
@@ -41,6 +41,7 @@ import (
 	xtime "github.com/m3db/m3/src/x/time"
 
 	"go.uber.org/zap"
+	"github.com/m3db/m3/src/dbnode/storage/namespace"
```

Comment:
nit: Surpised lint is picking this up, this should be with the imports above with all `github.com/m3db/m3/src...` imports.